src/basic/in-addr-util.c

```diff
@@ -241,6 +242,40 @@ int in_addr_to_string(int family, const union in_addr_union *u, char **ret) {
         return 0;
 }
 
+int in_addr_prefix_to_string(int family, const union in_addr_union *u, unsigned prefixlen, char **ret) {
+        _cleanup_free_ char *x = NULL;
+        char *p;
+        size_t l;
+
+        assert(u);
+        assert(ret);
+
+        if (family == AF_INET)
+                l = INET_ADDRSTRLEN + 3;
```

Comment:
+4, trailing NUL needs to be included, no?

Reply:
`INET_ADDRSTRLEN` is 16, thus it already includes trailing NUL. So, +3 is fine.